inc/Validation/Types.php

```diff
@@ -315,11 +316,15 @@ public static function is_serializable( array $type ): bool {
 	}
 
 	private static function is_type( string $type_name, array ...$types_to_check ): bool {
-		return array_reduce( $types_to_check, function ( bool $carry, array $type ) use ( $type_name ): bool {
+		foreach ( $types_to_check as $type ) {
```

Comment:
Same modification as what was done above.  It has a minor impact but every bit matters.

Reply:
This one makes sense to me, we get the benefit of an early return here.

Reply:
Ok, but this function is only called in two places. Once the array has a single member, the other it has two members